manifest.yml

```diff
@@ -1396,3 +1396,11 @@ dependencies:
   cf_stacks:
   - cflinuxfs4
   source_sha256: 4454dcd542031cdc3b839def90f5cad06ac2ed2cacddf3a209b3c0ab13904fc3
+- name: ruby
+  version: 3.0.5
+  uri: https://buildpacks.cloudfoundry.org/dependencies/ruby/ruby_3.0.5_linux_x64_cflinuxfs3_098393c3.tgz
```

Comment:
The major thing I need help with is that this Ruby dependency is compiled for `cflinuxfs3`, because we don't have a Ruby 3.0 dependency for `cflinuxfs4` (I believe because we couldn't compile it for cflinuxfs4 reasonably easily).

Trying to run the `compile-extensions` code with Ruby 3.1 produced the following errors:
```
/tmp/contents4188987394/deps/0/lib/ruby/3.1.0/psych/class_loader.rb:99:in `find': Tried to load unspecified class: Date (Psych::DisallowedClass)
   from /tmp/contents4188987394/deps/0/lib/ruby/3.1.0/psych/class_loader.rb:28:in `load'
   from (eval):2:in `date'
   from /tmp/contents4188987394/deps/0/lib/ruby/3.1.0/psych/scalar_scanner.rb:66:in `tokenize'
   from /tmp/contents4188987394/deps/0/lib/ruby/3.1.0/psych/visitors/to_ruby.rb:65:in `deserialize'
   from /tmp/contents4188987394/deps/0/lib/ruby/3.1.0/psych/visitors/to_ruby.rb:128:in `visit_Psych_Nodes_Scalar'
```


We can't just `apt-get install` the dependency because we don't have access during the build to do so.

Reply:
Strangely enough, using this `cflinuxfs3` dependency works, but feels wrong